From 57010ebb148b9b14e40140c7a8ce133a72128cf6 Mon Sep 17 00:00:00 2001
From: Robby Workman <rworkman@slackware.com>
Date: Thu, 18 Jul 2019 03:38:41 -0500
Subject: [PATCH] daemon/meson.build: define gvfs_rpath for libgvfsdaemon.so

On Slackware development branch with gvfs-1.40.2, I just noticed this:
  # ldd /usr/lib64/gvfs/libgvfsdaemon.so | grep "not found"
	libgvfscommon.so => not found

After some backtracking, it seems that this first occurred in the
switchover from autotools to meson in the 1.36.x --> 1.38.x bump.

Big thanks to Cogitri in #gnome/irc.gnome.org for the patience and
assistance with troubleshooting this.

Signed-off-by: Robby Workman <rworkman@slackware.com>
---
 daemon/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/daemon/meson.build b/daemon/meson.build
index 7f2608bd..0006f35e 100644
--- a/daemon/meson.build
+++ b/daemon/meson.build
@@ -121,6 +121,7 @@ libgvfsdaemon = shared_library(
   dependencies: deps,
   c_args: cflags,
   install: true,
+  install_rpath: gvfs_rpath,
   install_dir: gvfs_pkglibdir
 )
 
-- 
2.20.1

